docker/run_tests_in_docker.sh

```diff
@@ -25,7 +25,10 @@ if [ "$1" == "--shell" ]; then
 
     docker run \
            --rm \
-           --volume "$(pwd)":/app \
+           --volume "$(pwd)"/pontoon:/app/pontoon \
```

Comment:
Do you remember the reason why do we do this in the separate script instead of adding those volumes to docker-compose setup?
In the theory we could switch into something like:
`docker-compose run local/pontoon /bin/bash "${@:2}"`

If that's not possible, We could move those parameters into some variable and use it in both places.

Reply:
That was a copy and paste from Socorro. It is indeed inconsistent, I'll give it a thought and make it better.